install-project.sh

```diff
@@ -15,7 +15,7 @@ short_sha=${sha:0:7}
 modifiedslug_with_sha="${modifiedslug}-${short_sha}"
 modified_module=$(echo ${module} | sed 's?\./??g' | sed 's/\//+/g')
 
-if [[ -f "$AZ_BATCH_TASK_WORKING_DIR/$input_container/"${modifiedslug_with_sha}=${modified_module}".zip" ]]; then
+if [[ -f "$AZ_BATCH_TASK_WORKING_DIR/$input_container/projects/"${modifiedslug_with_sha}=${modified_module}".zip" ]]; then
```

Comment:
Is the idea of this if check that clone-project.sh would have already been unzipped and compiled/installed? If so, adding a comment here of the assumption that clone-project.sh is run and what it expects it to have done would be good.

Reply:
I added a comment like this "# if the project is installed before, then it doesn't install the project again and uses the already installed and zipped project"

Reply:
```# if the project is installed before, then it doesn't install the project again and uses the already installed and zipped project``` -> ```# We expect that clone-project.sh script is run before this script. If the project zip exists, then clone-project.sh should have unzipped an installed version of the project already. Therefore, we do not install the project again and we use the already installed and zipped project```

We should also add an echo here with a message like ``` echo "Project/sha/module zip already exist in input container and should be unzipped from clone-project.sh already. Skipping installation"